database.py

```diff
@@ -0,0 +1,20 @@
+import sqlite3
+
+import config
+
+# connection is a static variable that is shared across all instances of DB
+# so we reuse the same connection across all users of this class
+class Database():
```

Comment:
COMMENT: Im seeing these () things on all the class definitions, from what I understand, this gets around some issues that python 2 had, but its completely unneeded in python 3.8+ (the minimum discord.py python version)

Reply:
fixed